<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ScheduleAppointment</class>
 <widget class="QDialog" name="ScheduleAppointment">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>646</width>
    <height>439</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QCalendarWidget" name="calendarWidget">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>150</y>
     <width>200</width>
     <height>117</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_patientName">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>20</y>
     <width>71</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Patient Name:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_patientName">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>20</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>20</y>
     <width>91</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Phone Number:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_patienNumber">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>20</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QDateTimeEdit" name="dateTimeEdit">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>150</y>
     <width>194</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox_doctors">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>70</y>
     <width>111</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox_speciality">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>70</y>
     <width>111</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_doctor">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>70</y>
     <width>61</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Doctor:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_speciality">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>70</y>
     <width>61</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Speciality:</string>
   </property>
  </widget>
  <widget class="QPushButton" name="addToScheduleButton">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>240</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Add to Schedule</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
